docs/Other/Translating.md

```diff
@@ -47,6 +47,18 @@ Some entries have line breaks expressed as `\n`: Your translation can and in mos
 
 Chinese tutorial: 如果你是中国人，那么恭喜你运气不错！这里有Unciv中文开发者们专门为中文翻译工作者准备的（十分详尽）教程视频。：[(Video On Bilibili)](https://www.bilibili.com/video/BV1pY4y1u7WH/)
 
+## Adding a new language
+
+If any of the following steps are beyond your skillset, ask for help. All but the first two steps can be postponed.
+
+- You'll need to create a new file ('Create a new file' to the right of the folder name in the UI), and copy into it the contents of template.properties
```

Comment:
Maybe, personally when I started my previous translation attempt, these comments were very helpful. I would have translated a line and removed the comment. If I had skipped something. this would remind me of that. That was the target.